### ISSUE TITLE
UI: make additional steps

### ISSUE THREAD

**Issue by szszszsz** [Nitrokey/nitrokey-webupdate#16]
With #15 handling on Windows got complicated a bit. It would be nice to divide this step to two separate ones, so less information will be printed to user at once. Perhaps a confirmation button before starting (so the system's UI window will not occlude wizard) would be nice to have too. To review other steps.
